src/scheduler/data_types.h

```diff
@@ -1126,54 +1133,50 @@ struct config
 	bool non_prime_pre:1;
 	bool update_comments:1;	/* should we update comments or not */
 	bool prime_exempt_anytime_queues:1; /* backfill affects anytime queues */
-	bool assign_ssinodes:1;	/* assign the ssinodes resource */
 	bool preempt_starving:1;	/* once jobs become starving, it can preempt */
 	bool preempt_fairshare:1; /* normal jobs can preempt over usage jobs */
 	bool dont_preempt_starving:1; /* don't preempt staving jobs */
 	bool enforce_no_shares:1;	/* jobs with 0 shares don't run */
 	bool node_sort_unused:1;	/* node sorting by unused/assigned is used */
-	bool resv_conf_ignore:1;  /* if we want to ignore dedicated time when confirming reservations.  Move to enum if ever expanded */
-	bool allow_aoe_calendar:1;        /* allow jobs requesting aoe in calendar*/
+	bool resv_conf_ignore:1;	/* if we want to ignore dedicated time when confirming reservations.  Move to enum if ever expanded */
```

Comment:
do we need to the ":1" with bool?

Reply:
Yes, otherwise the size is undefined (likely 1 byte per bool)